cap/modules/experiments/utils/common.py

```diff
@@ -35,6 +35,13 @@
 from invenio_search.proxies import current_search_client as es
 
 
+def create_batches(items, chunk_size=1):
```

Comment:
can we make it accept a generator here?
can save loading milions of elements to the memory
also you can change here https://github.com/cernanalysispreservation/analysispreservation.cern.ch/blob/7329b0bf893e385c1f98b6fb27f72cdddc65a25d/cap/modules/experiments/tasks/cms.py#L111
as we don't need to care about unique elements anymore